src/qt/treectrl.cpp

```diff
@@ -125,6 +125,61 @@ class ImageState
 
 Q_DECLARE_METATYPE(TreeItemDataQt)
 
+// We use a custom selection model because we need to generate the
+// wxEVT_TREE_SEL_CHANGED event only if it is allowed. that is, only
```

Comment:
Nitpick of the day:
```suggestion
// wxEVT_TREE_SEL_CHANGED event only if it is allowed, that is, only
```

(alternatively, keep the period, but capitalize "That")

Reply:
forgot to update this